pkg/convert/pot.go

```diff
@@ -40,10 +48,22 @@ func ToPot(t model.Translation) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+func writePluralHeader(b *bytes.Buffer, t model.Translation) error {
```

Comment:
is the plural header the same for the whole translation file? Please add a description and URL to POT documentation to ease understanding.